site/demos/charts-dashboard/src/pages/Search.js

```diff
@@ -0,0 +1,99 @@
+import { ReactiveBase, ReactiveList, SearchBox, SelectedFilters } from '@appbaseio/reactivesearch';
+import { Col, Grid, Row, BackTop } from 'antd';
+import React from 'react';
+import CollapsibleFacets from '../components/CollapsibleFacets';
+import MobileFacets from '../components/MobileFacets';
+
+const { useBreakpoint } = Grid;
+
+export default function Search() {
+	const breakpointActive = useBreakpoint();
+
+	return (
+		<ReactiveBase
+			app="best-buy-dataset"
+			url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@appbase-demo-ansible-abxiydt-arc.searchbase.io"
+			enableAppbase
+		>
+			<Row>
+				<Row style={{ padding: 10, width: '100%' }}>
+					<SelectedFilters />
+				</Row>
+				<Row style={{ padding: 10, width: '100%' }}>
+					<SearchBox
+						dataField="albumTitle"
+						componentId="SearchBox"
+						style={{ width: '100%' }}
+						placeholder="Try searching for 'Tech Toys'"
+					/>
+				</Row>
+			</Row>
+			<Row>
+				<Col xs={24} md={8} style={{ padding: 10 }}>
+					{breakpointActive.md && <CollapsibleFacets />}
+					{breakpointActive.xs && <MobileFacets />}
+				</Col>
+				<Col xs={24} md={16} style={{ padding: 10 }}>
+					<ReactiveList
+						componentId="SearchResult"
+						dataField="original_title"
+						className="result-list-container"
+						from={0}
+						size={5}
+						renderItem={(data) => {
+							return (
+								<div
+									style={{ display: 'flex', padding: 20, background: '#fff' }}
```

Comment:
@SavvyShah Avoid using inline styles through out the app. You can create a global styles.css and put in your styles there.